### PR TITLE
Included close pair rejection in trigger, ppp trigger, fix in V0 checks

### PR DESCRIPTION
**Pull request by lauraser**
1. Close pair rejection included in trigger,
2. ppp trigger added,
3. obtaining V0 daughters from table had a bug, fixed in both FemtoDreamDetaDphiStar and FemtoDreamPairCleaner cases.

### CODE REVIEW

File: EventFiltering/PWGCF/CFFilter.cxx
Comment (victor-gonzalez) [inline EventFiltering/PWGCF/CFFilter.cxx:87]: You don't need a configurable for the magnetic field any longer
Have a look at https://github.com/AliceO2Group/O2Physics/blob/master/PWGCF/Tasks/correlations.cxx#L162

**Review by victor-gonzalez — APPROVED** [review]

File: EventFiltering/PWGCF/CFFilter.cxx
Comment (lauraser) [reply EventFiltering/PWGCF/CFFilter.cxx:87]: Thanks a lot for pointing to this example, Victor! Vale is implementing this now, but I want to merge the trigger code now as it is so that I can test it on hyperloop over the weekend. Next week we will include the magnetic field information appropriately.

**Review by jgrosseo — APPROVED** [review]

**Comment by jgrosseo** [comment]
Waiting for @mpuccio approval...

**Review by mpuccio — APPROVED** [review]

**Pull request merged**
